samples/nodejs_simple_sample/src/gateway_sample_lin.json

```diff
@@ -1,11 +1,20 @@
 {
+    "loaders": [
```

Comment:
Is the loaders section necessary? If this section is optional then I'd rather leave it out of the sample. We want to demonstrate a *minimal* working example...